blocks/v2-footer/v2-footer.js

```diff
@@ -0,0 +1,207 @@
+
+  const footerPath = cfg.footer || '/drafts/avinash/footer';
+  const resp = await fetch(`${footerPath}.plain.html`);
+  const html = await resp.text();
+  const footer = createElement('div', 'v2-footer-container');
+  footer.innerHTML = html;
+
+  // // adding the 'block' class and 'data-block-name'
+  // // needed to load blocks
+  const eloquaFormEl = footer.querySelector('.eloqua-form');
+  eloquaFormEl?.classList.add('block');
+  eloquaFormEl?.setAttribute('data-block-name', 'eloqua-form');
+
+  // transforming icons into font-awesome
+  footer.querySelectorAll('span.icon').forEach((icon) => {
+    const iconClass = icon.getAttribute('class').split(' ').find((el) => el.startsWith('icon-'));
+
+    if (iconClass) {
+      const iconName = iconClass.split('icon-')[1];
+      const link = icon.closest('a');
+      const social = iconName.replace('fa-', '');
+
+      link.ariaLabel = `${PLACEHOLDERS.visit.replace('$0', social)} ${
+        social !== 'youtube' ? PLACEHOLDERS.social : PLACEHOLDERS.channel}`;
+
+      icon.classList.remove('icon', iconClass);
+      icon.classList.add('fa', iconName, 'v2-footer-social-media');
+    }
+  });
+
+  const socialMediaSection = footer.querySelector('.fa-twitter, .fa-facebook, .fa-twitter, .fa-linkedin, .fa-instagram, .fa-youtube')?.closest('ul');
+  socialMediaSection?.classList.add('v2-footer-social-media-section');
+
+  const fourthHeader = [...footer.querySelectorAll('h3')].at(-1);
+  const [firstLinks, secondLinks, thirdLinks] = [...footer.querySelectorAll('h3 ~ ul')];
+
+  // secondLinks?.classList.add('footer-links-col-2');
+
+  // // creating the  logo link
+  const picture = footer.querySelector('picture');
+  let logoLink = null;
+
+  if (picture) {
+    logoLink = picture.closest('div').querySelector('p');
+    logoLink?.classList.add('footer-logo-link');
+
+    logoLink.innerHTML = '';
+    logoLink.append(picture);
+  }
+
+  const bottomLinksList = [...footer.querySelectorAll('ul')].at(-1);
+  bottomLinksList?.classList.add('v2-footer-bottom-section');
+
+  const formSection = footer.querySelector('.eloqua-form');
+  formSection?.classList.add('footer-form-section');
+
+  const footerTemplate = `
+    <div class="v2-footer-content">
+      <div class="v2-footer-main-content">
+        <div class="v2-footer-logo-section v2-logo">
+          ${logoLink?.outerHTML}
+        </div>
+        <div class="v2-footer-social-section v2-social">
+          ${socialMediaSection?.outerHTML}
+        </div>
+        <div class="v2-footer-links-section v2-content">
+          ${firstLinks?.outerHTML}
+          ${secondLinks?.outerHTML}
+          ${thirdLinks?.outerHTML}
+        </div>
+        <div class="v2-footer-form-section v2-form">
+            <div class="form v2-footer-form-header">
+              ${fourthHeader?.outerHTML}
+            </div>
+            <div class="form footer-form">
+              ${formSection?.outerHTML}
+            </div>
+          </div>
+      </div>
+      <hr />
```

Comment:
Replace the `<hr /> with styling for `.footer-bottom-section`

Reply:
Updated